DEV-Simple/ViewController.swift

```diff
@@ -187,7 +193,7 @@ class ViewController: UIViewController, WKNavigationDelegate {
         } else if isAuthLink(url) {
             return .allow
         } else if url.host != "dev.to" && navigationType.rawValue == 0 {
-            loadInBrowserView(url: url)
+            performSegue(withIdentifier: "showExternalPage", sender: url)
```

Comment:
Could you pull out this hard coded string to a global variable? Same with line 305